site/layouts/shortcodes/note.html

```diff
@@ -2,5 +2,5 @@
     <div class="note-icon">
         {{partial "svg/exclamation.svg" (dict "size" "20px" ) }}
     </div>
-    <div class="note-content">{{- .Inner -}}</div>
+    <div class="note-content"><div><strong>Note:</strong></div>{{- .Inner -}}</div>
```

Comment:
Would you please remove this? I like the idea, but I would rather allow to configure this. See https://github.com/mattermost/mattermost-developer-documentation/pull/344/commits/902d50c634e99b23bb75b2b8cd0f3f623532c558

Reply:
incorporated the changes